Makefile

```diff
@@ -30,14 +30,13 @@ godep: ## Ensure chainlink's go dependencies are installed.
 yarndep: ## Ensure the frontend's dependencies are installed.
 	yarn install
 
-build: godep gui $(SGX_BUILD_ENCLAVE) ## Build chainlink.
-	go build $(GOFLAGS) -o chainlink
+build: yarndep godep chainlink ## Build chainlink.
 
 install: godep gui ## Install chainlink
 	go install $(GOFLAGS)
 
-gui: yarndep ## Install GUI
-	CHAINLINK_VERSION="$(VERSION)@$(COMMIT_SHA)" yarn build
+gui: ## Install GUI
+	yarn build
```

Comment:
We need to set `CHAINLINK_VERSION` for the about page in react to have the correction version and sha:

```
CHAINLINK_VERSION="$(VERSION)@$(COMMIT_SHA)" yarn build
```

Reply:
All environment variables are exported by default, so there is no need to explicitly set them in the commands.

Reply:
ok, you've convinced me 👍